### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jimmywarting**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ silently dropped, rather than being passed through.
 ## Usage
 
 ```javascript
-const MuteStream = require('mute-stream')
+import MuteStream from 'mute-stream'
```
Comment (lukekarrys) [inline README.md:11]: this package is not authored with `esm` so I dont think it's a good idea to change the docs to use `import`.

can you revert this change?

**Review by lukekarrys — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jimmywarting) [reply README.md:11]: Why revert when you can just close?
This is just in the readme, and i think it's good that at least the documentation recommends ESM syntax...

Comment (lukekarrys) [reply README.md:11]: > Why revert when you can just close?

the other change in the PR is a good one 😄 

> This is just in the readme, and i think it's good that at least the documentation recommends ESM syntax

all the `npm` open source repos still use Common JS, so I'd like to stay consistent across all our repos w/r/t documentation matching the implementation.
